extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -28,12 +28,13 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		return true;
 	}
 
-	public static function render( $input, array $args, Parser $parser, PPFrame $frame ) {
+	public static function render( $input, array $args ) {
 		global $wgCityId;
 
-		$showLatest = empty( $args['mostrecent'] ) ? false : filter_var( $args['mostrecent'], FILTER_VALIDATE_BOOLEAN );
+		$showLatest = !empty( $args['mostrecent'] ) && filter_var( $args['mostrecent'], FILTER_VALIDATE_BOOLEAN );
 		$itemCount = empty( $args['size'] ) ? self::ITEMS_DEFAULT : intval( $args['size'] );
 		$columns = empty( $args['columns'] ) ? self::COLUMNS_DEFAULT : intval( $args['columns'] );
+		$category = empty( $args['category'] ) ? '' :  $args['category'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "category" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)